src/styles/utils.less

```diff
@@ -112,3 +111,64 @@
 .strikethru * {
   text-decoration: line-through;
 }
+
+[data-ironsworn-drop-type] {
```

Comment:
Might be good to move this to its own file, eh?

Reply:
good idea, will do